scout/utils/__init__.py

```diff
@@ -1 +1,14 @@
 # -*- coding: utf-8 -*-
+
+
+def call_safe(fun, unknown_var):
```

Comment:
Useful function! We have a few similar ones under scout/utils/convert.py. I would place this one there as well instead of here by itself

Reply:
Moved to `convert.py`.